### DISCUSSION TITLE
Let us change project names.

### DISCUSSION THREAD

**Discussion by ivanjeremic** [vercel/vercel#4255] [General] [answered]
Cloudways and other platforms allow us to change project names I think Vercel can learn from this and also add this feature, currently, we need to delete a project and create it again to get a new project name.

**Comment by coetry** [comment, marked as answer]
Hi @ivanjeremic we're well aware of this limitation and are planning to make this available soon. It's a little non-trivial due to an architectural decision we've made in the past regarding projects and so we are trying to make the transition in the most secure and flexible way.

**Reply by vicary** [reply]
Is there any updates on this?

**Comment by devoncmather** [comment]
+1

**Comment by lancemccluskey** [comment]
Are there any updates on this? 

@coetry 

![alt text](https://media.giphy.com/media/l0NwNrl4BtDD7JCx2/giphy.gif)

**Comment by maxsupera** [comment]
Yeah +1 -  this would be really great!
